acquia_lift.admin.inc

```diff
@@ -689,7 +689,7 @@ function acquia_lift_report($form, &$form_state, $agent_data) {
         );
```

Comment:
Sorry - part of this approach of a review is that I see things that aren't in the pull request... but related.
On line 652, I'm confused by the following code:

```
if (empty($audience) && !isset($option_set->targeting[$audience]['osid'])) {
    if (empty($audience_options)) {
      return array(
        'no_report' => array(
          '#markup'=> t('This personalization does not contain tests, no reports to show.')
        )
      );
    }
    $audience = key($audience_options);
  }
```

Should the other if statement actually be connected by || rather than &&?